fv/workload/workload.go

```diff
@@ -102,16 +104,35 @@ func Run(c *containers.Container, interfaceName, ip, port string) (w *Workload)
 	Expect(err).NotTo(HaveOccurred())
 
 	// Read the workload's namespace path, which it writes to its standard output.
-	namespacePath, err := bufio.NewReader(w.outPipe).ReadString('\n')
+	stdoutReader := bufio.NewReader(w.outPipe)
+	stderrReader := bufio.NewReader(w.errPipe)
+	namespacePath, err := stdoutReader.ReadString('\n')
 	Expect(err).NotTo(HaveOccurred())
 	w.namespacePath = strings.TrimSpace(namespacePath)
 
-	log.WithField("workload", w).Info("Workload now running")
+	go func() {
+		for {
+			line, err := stderrReader.ReadString('\n')
+			if err != nil {
+				log.WithError(err).Info("End of workload stderr")
+				return
+			}
+			log.Infof("Workload %s stderr: %s", name, strings.TrimSpace(string(line)))
+		}
+	}()
+	go func() {
+		for {
+			line, err := stdoutReader.ReadString('\n')
+			if err != nil {
+				log.WithError(err).Info("End of workload stdout")
+				return
+			}
+			log.Infof("Workload %s stdout: %s", name, strings.TrimSpace(string(line)))
+		}
+	}()
```

Comment:
This is really nice - but unfortunately it means that my much more hacky workload-output-logging code, in the Stop() method, needs to be deleted.  Do you mind doing that?